liota/dccs/iotcc.py

```diff
@@ -0,0 +1,248 @@
+# -*- coding: utf-8 -*-
```

Comment:
We are using the name Project ICE for all promotions in vmworld and other road shows. Why name it iotcc now?